packages/flutter/lib/src/widgets/text.dart

```diff
@@ -341,6 +346,9 @@ class Text extends StatelessWidget {
   /// closest enclosing [DefaultTextStyle].
   ///
   /// The [data] parameter must not be null.
+  /// 
+  /// The [overflow] property will be applied on a word basis when [softWrap] is true,
```

Comment:
same here.

Reply:
Sorry for late response. I tried to fix, please check them out!